packages/lzards-api-client/tests/test-lzards.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+
+const test = require('ava');
+const proxyquire = require('proxyquire');
+const sinon = require('sinon');
+const { randomId } = require('@cumulus/common/test-utils');
+
+const sandbox = sinon.createSandbox();
+const fakeGetToLzards = sandbox.stub();
+const fakeGetRequiredEnvVar = sandbox.stub();
+const fakeGetSecretString = sandbox.stub();
+const fakeGetLaunchpadToken = sandbox.stub();
+const lzards = proxyquire('../lzards', {
+  got: {
+    get: fakeGetToLzards,
+  },
+  '@cumulus/common/env': {
+    getRequiredEnvVar: fakeGetRequiredEnvVar,
+  },
+  '@cumulus/aws-client/SecretsManager': {
+    getSecretString: fakeGetSecretString,
+  },
+  '@cumulus/launchpad-auth': {
+    getLaunchpadToken: fakeGetLaunchpadToken,
+  },
+});
+
+const lzardsGetAuthToken = proxyquire('../lzards', {
```

Comment:
In general we've tried to avoid use of proxyquire (and less generally sinon, except in case of external dependencies like LZARDS calls) for maintainbility/patch behavior concerns. 

Could we instead use a test method parameter, e.g. 

```
xport async function sendGetRequestToLzards(
  {
    lzardsApiUri = process.env.lzards_api,
    searchParams,
    getAuthTokenFunction = getAuthToken,
  }: LzardsApiGetRequest
) {
```

typed like

```

export interface LzardsApiGetRequest {
  lzardsApiUri?: string,
  searchParams: object,
  getAuthTokenFunction: () => string,
}
```

?

Reply:
done

Reply:
Awesome, *many* thanks, that looks great. 

(Update at b587852479d0105f81eb4796ec81cf72e06d0b1d)